cmake/ExternalLibCEED.cmake

```
@@ -47,14 +47,16 @@ else()
 endif()
 
 # Build libCEED (always as a shared library)
+# STATIC=1 builds static lib
+# I think specifying anything for STATIC triggers static builds
 set(LIBCEED_OPTIONS
   "prefix=${CMAKE_INSTALL_PREFIX}"
   "LDFLAGS=${CMAKE_EXE_LINKER_FLAGS}"
   "CC=${CMAKE_C_COMPILER}"
   "CXX=${CMAKE_CXX_COMPILER}"
   "FC="
   "OPT=${LIBCEED_OPT_FLAGS}"
-  "STATIC="
+  # "STATIC=0"
```

```suggestion
  "STATIC="
```

Lets try and revert to original CMake...